domains/vinit.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "GharudxD",
+        "email": "vinitshahofficial@gmail.com"
+    },
+    "record": {
+        "TXT": "d663d938825df22848d6879cc0650b"
```

Comment:
TXT verification records must be paired with other records or in specialized files. See docs for more information. https://docs.is-a.dev/